src/interactive-window/editor-integration/cellhashprovider.ts

```diff
@@ -400,7 +343,7 @@ export class CellHashProvider implements ICellHashProvider {
         return Array.isArray(traceback) ? traceback.map(this.modifyTracebackFrame.bind(this)) : [];
     }
 
-    private findCellOffset(hashes: IRangedCellHash[] | undefined, codeLines: string): number | undefined {
+    private findCellOffset(hashes: ICellHash[] | undefined, codeLines: string): number | undefined {
```

Comment:
Can't comment farther down, but I thought the traceback stuff was moving out of this too?

Reply:
Yes, it wil.